recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+export DYLD_LIBRARY_PATH=$PREFIX/lib
```

Comment:
This might be problematic.  We should avoid DYLD_LIBRARY_PATH because in recent versions of OSX, they don't work unless SIP is disabled.